### DISCUSSION TITLE
RAG Issue: Hybrid Search Broken? No results or context provided to LLM

### DISCUSSION THREAD

**Discussion by Shadowfita** [open-webui/open-webui#7194] [Issues]
# Bug Report

## Installation Method

Latest docker image:main: 4bcc7f778d0372f5ba5fe91990b2a828a1970cd2b6ab102d06572de65b10a3a4

## Environment

- **Open WebUI Version:** 0.4.2
- **Ollama (if applicable):** 0.4.2

- **Operating System:** Ubuntu 20.04
- **Browser (if applicable):** Chrome

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

RAG request is processed and reranked accordingly.

## Actual Behavior:

LLM states there is no context to support the request when hybrid search is enabled with a reranking model.

## Description

**Bug Summary:**
LLM states there is no context to support the request when hybrid search is enabled with a reranking model.

## Reproduction Details

**Steps to Reproduce:**

Enable hybrid search with a reranker, resubmit documents for processing, ask a relevant question.

## Additional Information

Turning off hybrid search makes RAG work again.

**Comment by Shadowfita** [comment]
This seems to still be an issue in the latest release, 0.4.4.

**Reply by Koesn** [reply]
Yes, I experienced this too, no context sent LLM. Turning off hybrid make RAG work again.

Also when I set the OAI compatible endpoint, only embedding model works. Reranker seems still looking for model at SentenceTransformer repo on HF, and throw this error:
```
...
Repository Not Found for url: https://huggingface.co/api/models/sentence-transformers/Tugasi-Rerank/revision/main.
Please make sure you specified the correct `repo_id` and `repo_type`.
If you are trying to access a private or gated repo, make sure you are authenticated.
Invalid username or password.
...
```

I hope reranker following embedding model setting.
